Makefile

```diff
@@ -0,0 +1,80 @@
+#*******************************************************************************
+#   Ledger App
+#   (c) 2017 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  Modifications copyright (C) 2019 Hathor Labs
+#*******************************************************************************
+
+ifeq ($(BOLOS_SDK),)
+$(error Environment variable BOLOS_SDK is not set)
+endif
+include $(BOLOS_SDK)/Makefile.defines
+
+#########
+#  App  #
+#########
+
+APPNAME    = Hathor
+ICONNAME   = nanos_app_hathor.gif
```

Comment:
I think we should change the icon to have our new UI before sending to the "store". Maybe ignore it now if it's not so easy and create an issue to do it before publishing.

(I didn't know how to add a comment on the `.gif` file)

Reply:
We can do it on a separate PR. We also need to provide a larger icon for the Ledger manager: https://ledger.readthedocs.io/en/latest/additional/publishing_an_app.html#design-guidelines